official/utils/arg_parsers/parsers.py

```diff
@@ -176,6 +177,15 @@ def __init__(self, add_help=False, data_dir=True, model_dir=True,
           metavar="<HK>"
       )
 
+    if export_dir:
+      self.add_argument(
+          "--export_dir", "-ed",
+          help="[default: %(default)s] If set, a SavedModel serialization of "
+               "the model will be exported to this directory at the end of "
+               "training. See the README for more details and relevant links.",
+          metavar="<ED>"
+      )
```

Comment:
Should we provide a default directory if user provides none? Not sure if this feature is used frequently by users. If so, I think a default dir may be good.

Reply:
Considering that the functionality of SavedModel is currently limited to serving, I don't think this feature is frequently used.